### PR TITLE
Feature/forget password

### PR DESCRIPTION
**Pull request by Garima2505**
The candidate can resent the password using email address.

### CODE REVIEW

**Comment by ahanif19** [comment]
This looks good! Can we resolve the merge conflict before approving?

Also, I think that maybe this should redirect to a different page after you successfully reset your password. How about either the job listings page or a landing page for each user?

File: candidate_login/templates/candidate_login/password_reset_done.html [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{% extends "uplyft/base.html" %}
+{% block content %}
+    <div class="alert alert-info">
+        An email has been sent with instruction to reset your password
```
Comment (surajgovardhangaikwad) [inline candidate_login/templates/candidate_login/password_reset_done.html]: Just Nitpicking, but it should be "instructions"

File: candidate_login/views.py [hunk 2]
```diff
@@ -11,10 +11,10 @@ class CandidateLoginView(auth_views.LoginView):
 
 def login_success(request):
     if request.user.is_authenticated:
-        name = request.user.first_name
-        if name == "" or name is None:
-            name = request.user.username
-        messages.success(request, "Hi, " + name + "!")
+        email = request.user.email
+        if email == "" or email is None:
```
Comment (surajgovardhangaikwad) [inline candidate_login/views.py]: Shorthand for string checks can just be 
if not email:
(This is equivalent to -> if email == "" or email is None). 
Also, by convention, do None checks before empty. So it should be:
if email is None or email == "":
But then again, this is much better:
if not email:

File: teamone/settings.py [hunk 3]
```diff
@@ -147,3 +147,11 @@
 ACCOUNT_UNIQUE_EMAIL = True
 ACCOUNT_USER_MODEL_USERNAME_FIELD = None
 ACCOUNT_USERNAME_REQUIRED = False
+
+EMAIL_BACKEND = "django.core.mail.backends.smtp.EmailBackend"
+EMAIL_HOST = "smtp.gmail.com"
+EMAIL_PORT = 587
+EMAIL_USE_TLS = True
+EMAIL_HOST_USER = "uplyft1234@gmail.com"
+EMAIL_HOST_PASSWORD = "mwwymvtlbxnluvqx"
```
Comment (surajgovardhangaikwad) [inline teamone/settings.py]: This should not be pushed into the repo. This is confidential information

**Pull request merged**